libs/blocks/chart/list.js

```diff
@@ -93,23 +93,27 @@ const getCarouselHtml = (data, hexcode) => {
           aria-label="Next Chart"></button>
       </div>
       <div id="listChartCarousel-items"
-        class="carousel-items"
-        aria-live="polite">
+        class="carousel-items">
         ${carouselItems}
-      </div>
+        </div>
+        <div class="carousel-sr" aria-live="polite"></div>
     </section>`;
 };
 
 const showCarouselItem = (element, index) => {
   const carouselItems = element.querySelectorAll('.carousel-item');
+  const carouselSR = element.querySelector('.carousel-sr');
 
-  carouselItems?.forEach((carousel, idx) => {
+  carouselItems?.forEach((slide, idx) => {
     if (idx === index) {
-      carousel.classList.add('active');
+      slide.classList.add('active');
+      carouselSR.innerText = slide.querySelector('.title')?.innerText;
     } else {
-      carousel.classList.remove('active');
+      slide.classList.remove('active');
     }
   });
+
+  setTimeout(() => { carouselSR.innerText = ''; }, 5000);
```

Comment:
The timeout seems strange, and 5 secs seems a bit arbitrary. Is this the w3c recommendation?

Reply:
5 seconds is a bit arbitrary, but what was suggested by a11y, I couldn't find any w3c recommendation. 
With our use case aria-live wants to read the whole slide but we don't want to overload the reader with content, that's why its a separate div. https://gomakethings.com/how-and-why-to-use-aria-live/